test/functional/rpc_generateblock.py

```diff
@@ -51,37 +69,43 @@ def run_test(self):
         assert_equal(len(block['tx']), 1)
         assert_equal(block['tx'][0]['vout'][0]['scriptPubKey']['addresses'][0], combo_address)
 
-        # Generate 110 blocks to spend
-        node.generatetoaddress(110, address)
+        # Generate some blocks to spend
+        miniwallet.generate(100)
+        node.generate(100)
```

Comment:
You can keep this as just 10. It'll fix your premature spend error; 10 outputs is enough for this test.
```suggestion
        node.generate(10)
```